### PR TITLE
Disable experimental client roaming

### PR DESCRIPTION
**Pull request by ascendantlogic**
Disable the experimental client roaming feature to mitigate [this vulnerability](http://undeadly.org/cgi?action=article&sid=20160114142733)

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -21,9 +21,8 @@
 license          "Apache 2.0"
 description      "This cookbook installs and provides secure ssh and sshd configurations."
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.1.0"
+version          "1.2.0"
```
Comment (chris-rock) [inline metadata.rb]: can you remove the version bump from the PR? we do that separately

File: templates/default/openssh.conf.erb [hunk 2]
```diff
@@ -108,4 +108,7 @@ PermitLocalCommand no
 Compression yes
 
 #EscapeChar ~
-#VisualHostKey yes
\ No newline at end of file
+#VisualHostKey yes
+
+# http://undeadly.org/cgi?action=article&sid=20160114142733
+UseRoaming no
```
Comment (chris-rock) [inline templates/default/openssh.conf.erb]: would love to see an option to activate and deactivate roaming.

**Comment by chris-rock** [comment]
Thanks @ascendantlogic for adding this feature. I added some small comments

File: templates/default/openssh.conf.erb (hunk 2 above)
Comment (ascendantlogic) [inline templates/default/openssh.conf.erb]: Sure, I can do that. From what I understand this feature was never finished on the server-side, and was considered experimental on the client side.

File: metadata.rb (hunk 1 above)
Comment (ascendantlogic) [inline metadata.rb]: Yep, it's just habit from whenever I modify my own cookbooks :)

**Comment by ascendantlogic** [comment]
@chris-rock updated per your comments

**Comment by chris-rock** [comment]
Thanks @ascendantlogic

**Pull request merged**